packages/notifi-react-hooks/lib/utils/fetchDataImpl.ts

```diff
@@ -16,18 +15,18 @@ import {
   GetTopicsService,
   GetWebhookTargetsService,
   SmsTarget,
-  Source,
   SourceGroup,
```

Comment:
would be cool later to also update these from notifi-graphql too

Reply:
Np, Also created a relevant  ticket.